2023-09-15 17:06:11,026 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:07:11,097 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:08:11,205 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:09:11,310 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:10:11,422 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:11:11,520 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:12:11,616 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:13:11,710 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:14:11,816 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:15:12,556 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:16:12,910 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:17:13,017 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 616, in connect
    sock = socket.create_connection(
  File "/usr/lib/python3.10/socket.py", line 845, in create_connection
    raise err
  File "/usr/lib/python3.10/socket.py", line 833, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 75, in enqueue_events_for_site
    if is_scheduler_inactive():
  File "apps/frappe/frappe/utils/scheduler.py", line 114, in is_scheduler_inactive
    if is_scheduler_disabled(verbose=verbose):
  File "apps/frappe/frappe/utils/scheduler.py", line 127, in is_scheduler_disabled
    frappe.db.get_single_value("System Settings", "enable_scheduler")
  File "apps/frappe/frappe/database/database.py", line 786, in get_single_value
    ).run()
  File "apps/frappe/frappe/query_builder/utils.py", line 87, in execute_query
    result = frappe.db.sql(query, params, *args, **kwargs)  # nosemgrep
  File "apps/frappe/frappe/database/database.py", line 201, in sql
    self.connect()
  File "apps/frappe/frappe/database/database.py", line 115, in connect
    self._conn = self.get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 98, in get_connection
    conn = self._get_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 104, in _get_connection
    return self.create_connection()
  File "apps/frappe/frappe/database/mariadb/database.py", line 107, in create_connection
    return pymysql.connect(**self.get_connection_settings())
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 352, in __init__
    self.connect()
  File "env/lib/python3.10/site-packages/pymysql/connections.py", line 668, in connect
    raise exc
pymysql.err.OperationalError: (2003, "Can't connect to MySQL server on '127.0.0.1' ([Errno 111] Connection refused)")

2023-09-15 17:19:10,771 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:20:10,883 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:21:11,623 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:22:12,036 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:23:12,166 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:24:12,762 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:25:13,481 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:26:13,608 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:27:14,435 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:28:14,980 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:29:15,656 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:30:18,420 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:32:02,620 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:33:02,740 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:34:03,267 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:35:03,684 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:36:04,181 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:37:04,828 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:38:05,353 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:39:05,801 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:40:05,885 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:41:06,700 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:42:07,267 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:43:07,385 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:44:07,682 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:45:08,420 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:46:08,857 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:47:09,284 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:48:09,408 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:49:10,220 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:50:22,436 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:51:54,538 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:53:10,542 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:54:11,289 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:55:11,408 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:56:11,734 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:57:12,190 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:58:12,631 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 17:59:22,174 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:00:23,137 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:01:23,610 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:02:24,085 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:03:24,171 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:04:24,996 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:05:25,408 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:06:25,532 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:07:26,444 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:08:26,567 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:09:27,426 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:10:27,528 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:11:27,960 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:12:28,788 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:13:29,288 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:14:29,391 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:15:29,849 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:16:30,581 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:17:31,076 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:18:31,204 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:19:31,627 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:20:32,416 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:21:32,914 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:22:33,327 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:23:33,450 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:24:34,125 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:25:34,547 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:26:34,698 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:27:35,037 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:28:35,467 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:29:36,313 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:30:36,916 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:31:37,332 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:32:37,788 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:33:38,285 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:35:48,654 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

2023-09-15 18:36:49,159 ERROR scheduler Exception in Enqueue Events for Site erp.visitouriran.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 78, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 100, in enqueue_events
    frappe.get_doc("Scheduled Job Type", job_type.name).enqueue()
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 27, in enqueue
    if self.is_event_due() or force:
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 51, in is_event_due
    return self.get_next_execution() <= (current_time or now_datetime())
  File "apps/frappe/frappe/core/doctype/scheduled_job_type/scheduled_job_type.py", line 77, in get_next_execution
    "All": "0/" + str((frappe.get_conf().scheduler_interval or 240) // 60) + " * * * *",
TypeError: unsupported operand type(s) for //: 'str' and 'int'

